src/Array/Array.spec.ts

```diff
@@ -891,7 +891,7 @@ describe("Array", () => {
 
     describe("iter", () => {
         it("executes a function for each element of the array", () => {
-            const sideEffect = vi.fn<[], void>()
+            const sideEffect = vi.fn<() => void>()
```

Comment:
:question: vitest change?

Reply:
yep, this was a breaking change from version 1 -> 2